kyaml/yaml/fns.go

```diff
@@ -124,6 +124,100 @@ func (e ElementSetter) Filter(rn *RNode) (*RNode, error) {
 	return NewRNode(e.Element), nil
 }
 
+// ElementSetterList sets the value for an Element in an associative list.  ElementSetterList
```

Comment:
Please try to keep line length under 80 chars using linebreaks.
The first sentence here is perfect on a line by itself

Reply:
Due to the similarity, the comment here should describe the difference between
this and the ElementSetter struct on line 46 (old file)

Reply:
Also, fns_test.go needs to have a couple of regression tests for it.
If we are defining these things as public structs, we should cover them.

Maybe @Shell32-Natsu (last toucher of ElementSetter)
could first add some tests there for ElementSetter (in a different Pr) to establish a pattern.

Nothing fancy, just use the filter in a couple of cases.

Then this PR could follow that pattern.

Reply:
Likewise, again, to get the size of this PR down, the ElemenetSetterList and it's tests could be in a PR (or commit) by itself.  I prefer PR.

Reply:
opened #3176